### PR TITLE
Batch cert cmdlets (not for 0.9.9 release)

### PR DESCRIPTION
**Pull request by jasper-schneider**
New cmdlets for working with Batch certs.
I'm not aiming for the 0.9.9 release due tomorrow.  The next release is fine.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@jasper-schneider__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaschnei). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSJobReleaseTaskExecutionInformation.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            if ((omObject == null))
+            {
+                throw new System.ArgumentNullException("omObject");
+            }
+            this.omObject = omObject;
+        }
+        
+        public System.DateTime? EndTime
+        {
+            get
```
Comment (ogail) [inline src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSJobReleaseTaskExecutionInformation.cs]: nit: better to have on one line
```csharp
public System.DateTime? EndTime { get { return this.omObject.EndTime; } }
```

Comment (jasper-schneider) [inline src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSJobReleaseTaskExecutionInformation.cs]: This class is autogenerated.  I can see if there's a flag for line spacing preferences, but I'm not sure what CodeDom offers.

Comment (jasper-schneider) [inline src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSJobReleaseTaskExecutionInformation.cs]: Sorry, not really seeing any options for output formatting when defining properties: https://msdn.microsoft.com/en-us/library/system.codedom.codememberproperty%28v=vs.110%29.aspx

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Pull request merged**